cmd/handler-utils.go

```diff
@@ -126,51 +130,67 @@ func extractMetadataFromHeader(header http.Header) map[string]string {
 			metadata[cKey] = header.Get(key)
 		}
 	}
-	// Return.
+
+	// Success.
 	return metadata
 }
 
-// extractMetadataFromForm extracts metadata from Post Form.
-func extractMetadataFromForm(formValues map[string]string) map[string]string {
-	metadata := make(map[string]string)
-	// Save standard supported headers.
-	for _, supportedHeader := range supportedHeaders {
-		canonicalHeader := http.CanonicalHeaderKey(supportedHeader)
-		// Form field names are case insensitive, look for both canonical
-		// and non canonical entries.
-		if _, ok := formValues[canonicalHeader]; ok {
-			metadata[supportedHeader] = formValues[canonicalHeader]
-		} else if _, ok := formValues[supportedHeader]; ok {
-			metadata[supportedHeader] = formValues[canonicalHeader]
-		}
+// The Query string for the redirect URL the client is
+// redirected on successful upload.
+func redirectPostRawQuery(objInfo ObjectInfo) string {
```

Comment:
You could name this function as `GetPostRawQuery()` or `GetRedirectPostRawQuery()`